BE/src/main/java/org/team4/airbnb/domain/Review.java

```diff
@@ -0,0 +1,25 @@
+package org.team4.airbnb.domain;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+
+@Entity
+public class Review extends BaseCreated{
```

Comment:
`nit`
코드 포맷 사용을 인텔리제이에서 설정해주시는 것을 추천합니다. 네이버에서 제공하는 포맷이 있던데 저는 자바의 경우 이를 사용합니다.
```suggestion
public class Review extends BaseCreated {
```

Reply:
GoogleStyle을 둘 다 적용해서 개발하고 있는데요~ extendns BaseCreated 를 나중에 추가하면서 적용이 되지 않았던 것 같습니다 ^^;